### PR TITLE
dissable mac pipeline

### PR DESCRIPTION
**Pull request by vhvb1989**
Disabling MAC pipeline due to it is currently blocking other PRs

### CODE REVIEW

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -24,10 +24,10 @@ jobs:
         VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
         CMAKE_GENERATOR: 'Visual Studio 16 2019'
         CMAKE_GENERATOR_PLATFORM: x64
-      MacOS_x64:
-       vm.image: 'macOS-10.14'
-       vcpkg.deps: ''
-       VCPKG_DEFAULT_TRIPLET: 'x64-osx'
+      #MacOS_x64:
```
Comment (danieljurek) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml]: This job passed and doesn't need to be disabled. It'll give us _some_ clang coverage.

Comment (vhvb1989) [reply eng/pipelines/templates/jobs/archetype-sdk-client.yml]: yeah, ok, Adding it back.
thanks

**Comment by antkmsft** [comment]
Can you provide more context? Which PRs is it blocking? I recently enabled extra warnings for Clang, if that's for that reason, they are useful.

**Comment by gilbertw** [comment]
Why is MAC build failing?  When would it be re-enabled.  I prefer we fix the MAC build.

Note, if the blocked checkin is critical and the MAC build failure is unrelated.  We can get a repo admin to commit

**Comment by vhvb1989** [comment]
@antkmsft @rickle-msft 
All PRs gets blocked if any pipeline is falling
The error is when trying to bootstrap VCPKG in Mac.  @danieljurek  is already looking into it.
Looks like mac images were updated yesterday and GCC version changed

This is not related to our code at all.

I was trying to merge this : https://github.com/Azure/azure-sdk-for-c/pull/482

And also iot guys asked me about it
I started a thread in C channel for this.

Please apologies for the rush on this

**Pull request merged**